internal/generator/vector/pipelines.go

```diff
@@ -33,6 +33,7 @@ func Pipelines(spec *logging.ClusterLogForwarderSpec, op generator.Options) []ge
 parsed, err = parse_json(.message)
 if err == null {
   .structured = parsed
+  del(.message)
```

Comment:
Can we defer JSON parsing till the end of the line (the output) so that the output can decide if it wants JSON parsing or not? We could mark the record here as .json_parse=true, and then in the output do the parse 
`if .json_parse && i_want_structured_data`

Reply:
This fix is for fluent only.  I have not investigated if we are breaking vector... but maybe i should :)

Reply:
Agreed that we shouldn't parse multiple times, so variation of this - parse at the pipeline and discard message or structured at the output?